powerauth-webflow-authentication-form/src/main/java/io/getlime/security/powerauth/lib/webflow/authentication/method/form/service/PasswordEncryptionService.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2019 Wultra s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.getlime.security.powerauth.lib.webflow.authentication.method.form.service;
+
+import com.google.common.io.BaseEncoding;
+import io.getlime.security.powerauth.crypto.lib.config.PowerAuthConfiguration;
+import io.getlime.security.powerauth.crypto.lib.generator.KeyGenerator;
+import io.getlime.security.powerauth.crypto.lib.util.AESEncryptionUtils;
+import io.getlime.security.powerauth.lib.dataadapter.model.enumeration.AuthenticationType;
+import io.getlime.security.powerauth.lib.webflow.authentication.configuration.WebFlowServicesConfiguration;
+import io.getlime.security.powerauth.provider.CryptoProviderUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import javax.crypto.SecretKey;
+import java.nio.charset.StandardCharsets;
+
+/**
+ * Service for encryption of user passwords.
+ *
+ * @author Roman Strobl, roman.strobl@wultra.com
+ */
+@Service
+public class PasswordEncryptionService {
+
+    private static final Logger logger = LoggerFactory.getLogger(PasswordEncryptionService.class);
+
+    private final WebFlowServicesConfiguration configuration;
+
+    private final AESEncryptionUtils aesEncryptionUtils = new AESEncryptionUtils();
```

Comment:
... I like calling this instance just `aes` - it makes the code more readable later.